### PR TITLE
Fix/magic block

### PR DESCRIPTION
**Pull request by dabasov**
## Fixes

magic-block generation is now up to date with mainnet requirements

### CODE REVIEW

File: code/go/0chain.net/chaincore/block/magicBlock/main.go [hunk 1]
```diff
@@ -42,14 +43,14 @@ type cmdMagicBlock struct {
 
 var (
 	defaultTokenSize int64 = 10000000000
-	//rootPath               = "/config"
-	rootPath = "/Users/dabasov/Projects/0chain_others/magic-block/docker.local/config"
+	rootPath               = "/config"
+	//rootPath = "/Users/dabasov/Projects/0chain_others/magic-block/docker.local/config"
```
Comment (peterlimg) [inline code/go/0chain.net/chaincore/block/magicBlock/main.go]: Remove the commented code

File: docker.local/build.magicBlock/Dockerfile [hunk 2]
```diff
@@ -2,19 +2,25 @@ FROM zchain_build_base as magic_block_build
 ENV SRC_DIR=/0chain
 ENV GO111MODULE=on
 
-COPY ./code/go/0chain.net $SRC_DIR/go/0chain.net
+#COPY ./code/go/0chain.net $SRC_DIR/go/0chain.net
```
Comment (peterlimg) [inline docker.local/build.magicBlock/Dockerfile]: Remove commented code

**Review by peterlimg — CHANGES_REQUESTED** [review]
The zip files exist in both magicBlock/input folder and the `one` and `two` folders, should any of them be removed from the commits?

**Comment by dabasov** [comment]
> The zip files exist in both magicBlock/input folder and the `one` and `two` folders, should any of them be removed from the commits?

cleaned

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**